CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 Breaking changes:
 - Remove the old `_converse.BootstrapModal` in favor of `_converse.BaseModal` which is a web component.
+- #fixes #3133 changes _controlbox.scss to fix the overflow issue
```

Comment:
This isn't a breaking change, so shouldn't go under the `Breaking changes` heading.

Reply:
okay i'm new here so i don't know where i put those changes list. can you tell me whare i write these changes

Reply:
Just move it up above the `Breaking changes` heading (but below `# 11.0.0 (Unreleased)`).